scan-payment/src/main/java/com/getbouncer/scan/payment/ml/yolo/yolo.kt

```diff
@@ -0,0 +1,68 @@
+package com.getbouncer.scan.payment.ml.yolo
+
+import android.graphics.RectF
+import android.util.Size
+import com.getbouncer.scan.framework.util.indexOfMax
+import com.getbouncer.scan.payment.ml.ssd.DetectionBox
+import kotlin.math.exp
+
+fun sigmoid(x: Float): Float = (1.0f / (1.0f + exp((-x))))
+
+fun softmax(a: FloatArray): FloatArray {
+    var sum = 0f
+    for (f in a) {
+        sum += exp(f)
+    }
+    return a.map { exp(it) / sum }.toFloatArray()
+}
```

Comment:
consider `ClassifierScores.softMax2D`, which performs this same function but a little faster.